peekingduck/pipeline/nodes/draw/blur_bbox.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2021 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Blur area bounded by bounding boxes over detected object
+"""
+
+from typing import Any, Dict, List
+import numpy as np
+from scipy.ndimage import gaussian_filter
+from peekingduck.pipeline.nodes.node import AbstractNode
+
+
+class Node(AbstractNode):  # pylint: disable=too-few-public-methods, duplicate-code
+    """Blur area bounded by bounding boxes on image.
+
+    The blur_bbox node blur the areas of the image
+    bounded by the bounding boxes output from an object detection model
+
+    Inputs:
+
+        |img|
+
+        |bboxes|
+
+    Outputs:
+        |img|
+
+    Configs:
+        None.
+    """
+
+    def __init__(self, config: Dict[str, Any] = None, **kwargs: Any) -> None:
+        super().__init__(config, node_path=__name__, **kwargs)
+
+    # pylint: disable=duplicate-code
```

Comment:
Did this line of code manage to suppress the warning? If no, remove this line.

Reply:
the pylint: disable=duplicate-code comment is removed, as it does not work